doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -14086,6 +14107,375 @@ AirConditioner:VariableRefrigerantFlow,
+
+#### Field: Outdoor Unit Reference Subcooling 
+
+This numeric field defines the reference subcooling degrees of the outdoor unit. If this field is blank, the default value of 3.0ºC is used. 
+
+#### Field: Refrigerant Temperature Control Algorithm for Indoor Unit
+
+This alpha field specifies the algorithm for the refrigerant temperature control. Two choices are available: ConstantTemp or VariableTemp. The indoor unit evaporating temperature at cooling mode or condensing temperature at heating are fixed in the ConstantTemp algorithm, while in VariableTemp algorithm they can be varied.
+
+#### Field: Reference Evaporating Temperature for Indoor Unit 
+
+This numeric field defines the reference evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is ConstantTemp. If this field is blank, the default value of 6.0ºC is used.
+
+#### Field: Reference Condensing Temperature for Indoor Unit 
+
+This numeric field defines the reference condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is ConstantTemp. If this field is blank, the default value of 44.0ºC is used.
+
+#### Field: Variable Evaporating Temperature Minimum for Indoor Unit
+
+This numeric field defines the minimum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 4.0ºC is used.
+
+#### Field: Variable Evaporating Temperature Maximum for Indoor Unit
+
+This numeric field defines the maximum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 13.0ºC is used.
+
+#### Field: Variable Condensing Temperature Minimum for Indoor Unit
+
+This numeric field defines the minimum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 42.0ºC is used.
+
+#### Field: Variable Condensing Temperature Maximum for Indoor Unit
+
+This numeric field defines the maximum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 46.0ºC is used.
+
+#### Field: Outdoor Unit Fan Object Type
+
+This alpha field defines the type of fan in the VRF outdoor unit. The only valid choices are *Fan:OnOff* and *Fan:ConstantVolume*. The input requirements for these fan objects are described elsewhere in this document.
+
```

Comment:
I think for now that would be best, but it needs to be autosizable or 
scaleable somehow.  Other DX equipment just includes this in the overall 
COP, so you could just drop it altogether for now and indicate that the 
COP includes the outdoor fan power.  It's fine to have it separate, but 
should be W per compressor like the crankcase heat, or W per capacity 
unit.  I don't even see a total capacity for the outdoor unit?

On 8/27/2015 4:34 PM, Rongpeng Zhang wrote:
>
> In doc/src/docs/InputOutputReference/01c-InputOutputReference.md 
> <https://github.com/NREL/EnergyPlus/pull/5026#discussion_r38148968>:
>
> > +#### Field: Variable Evaporating Temperature Maximum for Indoor Unit
> > +
> > +This numeric field defines the maximum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 13.0ºC is used.
> > +
> > +#### Field: Variable Condensing Temperature Minimum for Indoor Unit
> > +
> > +This numeric field defines the minimum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 42.0ºC is used.
> > +
> > +#### Field: Variable Condensing Temperature Maximum for Indoor Unit
> > +
> > +This numeric field defines the maximum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 46.0ºC is used.
> > +
> > +#### Field: Outdoor Unit Fan Object Type
> > +
> > +This alpha field defines the type of fan in the VRF outdoor unit. The only valid choices are *Fan:OnOff* and *Fan:ConstantVolume*. The input requirements for these fan objects are described elsewhere in this document.
> > +
>
> @mjwitte <https://github.com/mjwitte>
>
> I think I can move the fan information back to VRF object, as what it 
> was originally designed. It is just one field:
> 170, !- Rated Condenser Fan Power {W}
>
> I think this value can be obtained from the mfr data. If you think 
> this is fine, I will do that and get this branch merged.
>
> I moved the fan information from VRF object to fan object because I 
> thought it may benefit the future development/maintenance if the 
> system would add more control logic on the OU fan operation, or the 
> fan is improved from constant speed to variable speed. For now, it is 
> not really necessary to separate it into a fan object, since it is 
> constant speed, and is not located in any air loops. Maybe we can have 
> further discuss this later after this busy season. How do you think 
> about it?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NREL/EnergyPlus/pull/5026/files#r38148968>.
>